packages/serverpod/lib/src/relic/routes/static_directory.dart

```diff
@@ -27,8 +27,23 @@ class RouteStaticDirectory extends Route {
   /// The path to the directory to serve static files from.
   final String? basePath;
 
+  /// A regular expression that will be used to determine if a path should not
+  /// be cached.
+  late final RegExp? noCachePathRegexp;
```

Comment:
Suggestion: I think we can keep this private I see no value in exposing this variable.

Reply:
Done